bwl_loot.json

```diff
@@ -338,7 +338,7 @@
   {
     "id": 19358,
     "name": "Draconic Maul",
-    "role": "Warrior = Enhance-Shaman",
+    "role": "Warrior = Enhance-Shaman = Feral-Druid",
```

Comment:
we dont have a feral so our feral prio is pretty inconsistent, dont think this is controversial